assets/model_monitoring/components/tests/unit/test_numerical_data_drift_metrics.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""This file contains unit tests for the Data Drift compute metrics component component."""
+
+from data_drift_compute_metrics.io_utils import init_spark
+from data_drift_compute_metrics.numerical_data_drift_metrics import compute_numerical_data_drift_measures_tests
+import pandas as pd
+import pytest
+import numpy as np
+import unittest
+
+test_cases = [
+                [100_000, 100_000, 50, 50, 0],
```

Comment:
This will only be 0 when x==y. In the current test, the distributions are not the exactly the same. Instead, you are you are creating two different distributions x and y with the same parameters.

Reply:
Yeah, you are right. Didn't realize the randomness here. Let me make a separate test for this case.

Reply:
Updated